docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -92,7 +96,7 @@ Destructive Tests
 These tests are allowed to install and remove some trivial packages.  You will likely want to devote these
 to a virtual environment, such as Docker.  They won't reformat your filesystem::
 
-    ansible-test integration --docker fedora29 -v destructive/
+    ansible-test integration destructive/ --docker fedora32 --docker-no-pull
```

Comment:
The `--docker-no-pull` option isn't needed here.

Reply:
removed